backend/apid/graphql/dataloader.go

```diff
@@ -326,10 +335,9 @@ func getLoader(ctx context.Context, loaderKey key) (*dataloader.Loader, error) {
 // When resolving a field, GraphQL does not consider the absence of a value an
 // error; as such we omit the error if the API client returns Permission denied.
 func handleListErr(err error) error {
-	if apiErr, ok := err.(client.APIError); ok {
-		if apiErr.Code == uint32(actions.PermissionDenied) {
-			return nil
-		}
+	if err == authorization.ErrUnauthorized {
```

Comment:
<img src="https://user-images.githubusercontent.com/194892/63125210-f2ef1b00-bf61-11e9-8cae-d98b91efd5eb.png" width="400" />

I'm frequently seeing authorization errors when running the web app. I suspect this is because we're wrapping upstream errors in the client interface, eg. https://github.com/sensu/sensu-go/blob/94b0e1147b93422705dcd93847feea3ae9c3cc57/backend/api/mutator.go#L39-L41

Reply:
I got errors when I ran the frontend too, but I tried again with incognito/private mode, and they went away, so I wrote them off as localstorage problems. Any idea why?